src/SprykerEco/Zed/ComputopApi/ComputopApiDependencyProvider.php

```diff
@@ -25,13 +25,13 @@ public function provideBusinessLayerDependencies(Container $container): Containe
     {
         $container = parent::provideBusinessLayerDependencies($container);
 
-        $container[static::SERVICE_COMPUTOP_API] = function () use ($container) {
+        $container->set(static::SERVICE_COMPUTOP_API, function () use ($container) {
```

Comment:
* In order to use setters, adjust `composer.json` with support of the `"spryker/kernel": "^3.30.0".

Reply:
Done